encoding/thrift/thriftrw-plugin-yarpc/uberfx_template.go

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package main
+
+const uberfxServerTemplate = `
+// Code generated by thriftrw-plugin-yarpc and thriftrw-plugin-fx
+// @generated
+
+<$pkgname := printf "%sserver" (lower .Service.Name)>
+package <$pkgname>
+
+<$fx        := import "go.uber.org/fx">
+<$rpc       := import "go.uber.org/fx/modules/rpc">
+<$service   := import "go.uber.org/fx/service">
```

Comment:
Thanks. I sent an update with the addressed feedback of not using separate template and injected variables/imports.